perma_web/perma/tasks.py

```diff
@@ -973,33 +948,29 @@ def _proxy_request(self):
             print("Finished fetching URL.")
 
             # Get a fresh copy of the page's metadata, if possible
-            with browser_running(link, browser):
-                inc_progress(capture_job, 1, "Retrieving page metadata (after onload)")
-                add_thread(thread_list, get_metadata, args=(
-                    page_metadata,
-                    link,
-                    browser
-                ))
+            dom_tree = get_dom_tree(browser)
+            get_metadata(page_metadata, dom_tree)
```

Comment:
Intriguingly, this is grabbing the DOM of the Forbes welcome page the majority of the time. Maybe after all these changes, the 25s delay needs to be extended by a second or two.....